classes/PvXCode.php

```diff
@@ -24,12 +24,12 @@ class PvXCode {
 	public static function parserRender( string $input, array $args, Parser $parser, string $frame ) {
 		$parser->getOutput()->addModuleStyles( 'ext.PvXCode.css' );
 		$parser->getOutput()->addModules( 'ext.PvXCode.js' );
-		$title = $parser->getTitle()->getText();
+		$title = $parser->getPage()->getText();
```

Comment:
The `PageReference` interface does not define a method `getText()`. It may instead be necessary to create a `LinkTarget` first:

```php
$title = TitleValue::newFromPage( $parser->getPage() );
$text = $page->getText();
```